packages/forms-web-app/__tests__/unit/routes/before-you-start/index.test.js

```diff
@@ -0,0 +1,18 @@
+const { use } = require('../router-mock');
+
+describe('routes/before-you-start/index', () => {
+  beforeEach(() => {
+    jest.resetModules();
+
+    // eslint-disable-next-line global-require
+    require('../../../../src/routes/before-you-start');
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('should define the expected routes', () => {
+    expect(use.mock.calls.length).toBe(1);
```

Comment:
We may another expectation that checks for call to specific router
```suggestion
    expect(use.mock.calls.length).toBe(1);
    expect(use).toHaveBeenCalledWith('/', localPlanningDepartmentRouter);
```

Reply:
Edited.

Reply:
Leaving this test as it is to keep consistency with other sections.